main.py

```diff
@@ -246,8 +243,8 @@ def pack(self):
         return pack([{self.tag: 1}, {self.bootstrap_method_attr_index: 2}, {self.name_and_type_index: 2}])
 
 
-for i in range(constant_pool_count - 1):
-    tag, classData = nextBytes(1, classData)
+for i in range(len(constant_pool_count) - 1):
+    tag, classData = nextBytes(1, classData, bytes)

```

Comment:
This causes the value not to be present in the `lengthTable` leading to an error